### PR TITLE
📦 Home page: revision from InVision

### PR DESCRIPTION
**Pull request by mamadOuologuem**

### CODE REVIEW

File: src/components/styledComponents.js [hunk 1]
```diff
@@ -145,16 +146,32 @@ export const defaultTheme = createMuiTheme({
...
+    MuiIconButton: {
+      root: {
+        padding: '8px 0px 8px 12px',
+      },
+    },
+    MuiMenuItem: {
+      root: {
+        width: '100%',
+        '&:hover': {
+          backgroundColor: `${colors.orange}2f`,
```
Comment (bencpeters) [inline src/components/styledComponents.js]: What is the `2f` doing here?

Comment (mamadOuologuem) [reply src/components/styledComponents.js]: It is the transparency.
I wasn't sure if it is useful to add a new color for that single used

Comment (bencpeters) [reply src/components/styledComponents.js]: Ah, I didn't realize you can add on another pair for rgba values in that format... that's cool!

Comment (mamadOuologuem) [reply src/components/styledComponents.js]: Other comment ?

Comment (bencpeters) [reply src/components/styledComponents.js]: Looks good to merge to me!

**Pull request merged**